modules/core/client/directives/pagination.client.directive.html

```diff
@@ -0,0 +1,7 @@
+<div class="list-group">
+    <div data-ng-repeat="item in pagedItems">
```

Comment:
http://www.williambrownstreet.net/blog/2013/07/angularjs-my-solution-to-the-ng-repeat-performance-problem/
http://www.williambrownstreet.net/blog/2013/07/angularjs-my-solution-to-the-ng-repeat-performance-problem/